src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs

```diff
@@ -66,35 +67,50 @@ public HashSet<int> Start(ApplicationData applicationData)
             return selectedIndexes;
         }
 
+        private void Accept(){
+            Application.RequestStop();
+        }
+
+        private void Close(){
+            _cancelled = true;
+            Application.RequestStop();
+        }
+
         private Window AddTopLevelWindow()
         {
             // Creates the top-level window to show
             var win = new Window(_applicationData.Title)
             {
                 X = 0,
-                Y = 1, // Leave one row for the toplevel menu
+                Y = 0, 
                 // By using Dim.Fill(), it will automatically resize without manual intervention
                 Width = Dim.Fill(),
                 Height = Dim.Fill()
             };
 
+            // win.Accept += () => Accept();
+            // win.Close += () => Close();
+
             Application.Top.Add(win);
             return win;
         }
 
-        private void AddMenu()
+        private void AddStatusBar()
         {
-            var menu = new MenuBar(new MenuBarItem []
-            {
-                new MenuBarItem("_Actions (F9)", 
-                    new MenuItem []
+            var statusBar = new StatusBar(
+                    _applicationData.OutputMode != OutputModeOption.None
+                    ? new StatusItem []
+                    {
+                        new StatusItem(Key.Enter, "~ENTER~ Accept", () => Accept()),
```

Comment:
Are these clickable? (I believe you answered already but want to confirm)

Reply:
No. Not yet anyway. See https://github.com/migueldeicaza/gui.cs/issues/360